UI/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
add much more to this if you are planning on keeping your .idea and venv files here locally

Reply:
fix this before I review further otherwise its impossible to find files in this 2500 files added